### PR TITLE
add release instructions for the firmware

### PR DESCRIPTION
**Pull request by knmcguire**

### CODE REVIEW

File: release.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## 3. Do the Crazyflie testing
+1. Do all the Crazyflie release testing
+    * If all is good, continue!
+    * If there is something wrong, fix the issues, and start again to generate a new version of the release candidate for both the firmwares and again of the full crazyflie-release
+
+## 4. Preparing the real release
+1. Now getting ready for the real release: set the real release tag (like '2022.03') on the crazyflie-firmware and crazyflie2-nrf firmware,
+1. Use the release actions with that tag
+1. Go to the draft release and input the generate thed release notes by  `tb ghrn bitcraze/crazyflie-firmware 2022.03`
```
Comment (krichardsson) [inline release.md]: Typo

File: release.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+Releases are done from Github Actions.
+The overall process is to first tag the repo and then build it on Github Actions. The build will run tests, build binaries, generate a release in github and upload binaries. Continue to read about the crazyflie-release project for handling of full fw releases and release candidates.
+
+## 1. Preparing the candidate release
+1. Tag the crazyflie-firmware and the crazyflie2-nrf-firmware github repo with a release candidate tag in the form of “2022.03-rc1”
+1. In github: Collect issues of both repositories in the “next-release” milestone. 
+1. In github: Rename the milestones to the real release, for instance “2022.03”
+1. In github actions on both firmware's repos: 
+   * select the 'release' flow. Press the dow narrow on 'run workflow'
```
Comment (krichardsson) [inline release.md]: typo

File: release.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+   * select the release candidate tag (see screenshot under)
+   * press the green button 'Run Workflow'. 
+1. Wait until workflows has been finalized
+1. Make to two pre-releases of the two repositories public
+
+![](action_screenshot.png)
+
+## 2. Make a candidate release firmware zip folder
+1. Locally clone this repository (crazyflie-release)
```
Comment (krichardsson) [inline release.md]: I would suggest to remove this part. As we move the build functionality to github, I think we should stop using the toolbelt release functionality (and remove it at some point). Just build the release candidate (in crazyflie-release) and use that for testing.

**Review by krichardsson — CHANGES_REQUESTED** [review]
Looking good! Some minor comments bellow. 

One option would be to put this in a docs directory to make it compatible with the web. Not sure there is any value in publishing it on the web as this mainly is an instruction for Bitcraze.

**Comment by knmcguire** [comment]
Thanks kristoffer! I moved it to a docs folder now as well. 

Also I updated the doc to not use the toolbelt locally but just to generate the zipfiles directly with actions. People can just download it from the draft if they need to test it first. 

I'll merge it so that I can remove the old instructions

**Pull request merged**